t/kubernetes/discovery/kubernetes3.t

```diff
@@ -332,6 +339,8 @@ POST /operators
 ]
 --- more_headers
 Content-type: application/json
+--- response_body_like
+.*"endpoints".*
```

Comment:
```
--- response_body_like
.*"endpoints".*
```

this doesn't seem sufficient to show that the feature works fine. The verification should be more descriptive.